theme/tutorial.less

```diff
@@ -491,22 +497,95 @@ code.lang-filterblocks {
     color: @white;
     z-index: @simulatorZIndex;
 
-    #workspaceHeader {
-        flex: 1;
+    #headerToolbar {
+        position: absolute;
+        right: 0;
     }
 
     #flyoutHeader {
         box-sizing: content-box;
         overflow: hidden;
         white-space: nowrap;
         text-overflow: ellipsis;
+
+        #flyoutHeaderTitle{
+            padding-left: 1.2rem;
+            padding-top: .8rem;
+        }
     }
 
     div {
         display: inline-block;
         text-align: center;
         line-height: 2rem;
-        font-weight: 800;
+    }
+}
+
+.flyoutOnly {
+    #headers {
+        height: @workspaceHeaderHeight;
+
+        div {
+            display: inline-flex;
+        }
+    }
+}
+
+/*******************************
+        Sidebar Tutorial
+*******************************/
+.flyoutOnly.sidebarTutorial.tutorial {
+    #maineditor{
+        left: 33.3%;
+    }
+
+    #maineditor > .full-abs {
+        top: 0;
+    }
+
+    #maineditor > #blocksArea {
+        top: 3rem;
+    }
+
+    #sidebar {
+        width: 33.3%;
+        height: 100%;
+        background-color: @sidebarTutorialColor;
+        color: #3c3c3c;
+        font-family: @docsPageFont;
+        padding: @sidebarPadding / 2 @sidebarPadding;
+
+        .tutorialTitle {
+            font-size: 1.5rem;
```

Comment:
small: might be useful to move these font sizes to a less variable as well (@sidebarTutorialHeaderSize or something) since they seem to be used in several places